src/components/lib/Steps.tsx

```diff
@@ -53,27 +59,36 @@ export abstract class Steps {
 
   public static isValid(form: IForm, questionId: string): boolean {
     const q = form.responses.find((a) => a?.id === questionId);
-    if (!q) return false;
-    const answers = values(q.answers);
+    let ret = true;
+    if (!q) {
+      ret = false;
+    }
```

Comment:
This could probably be "simplified" to 
```js
let ret = q;
```
(assuming I got the boolean transformations correct) but maybe it's less readable.